subprojects/launcher/src/main/java/org/gradle/launcher/daemon/logging/DaemonMessages.java

```diff
@@ -34,4 +34,5 @@
     public static final String REMOVING_DAEMON_ADDRESS_ON_FAILURE = "Removing daemon from the registry due to communication failure. Daemon information: ";
     public static final String UNABLE_TO_STOP_DAEMON = "Unable to stop one of the daemons. The daemon may have crashed.";
     public static final String WAITING_ON_CANCELED = "Waiting for daemons with canceled builds to become available";
+    public static final String UNABLE_TO_NOTIFY_DAEMON_ABOUT_FILE_CHANGES = "Unable to notify the daemon about file changes.";
```

Comment:
Nit: this should say "path" changes, no?

Reply:
I removed the constant since it is not used anymore - we now throw the exception.